icepyx/core/harmony.py

```diff
@@ -1,13 +1,14 @@
 from typing import Any
 
-import requests
-
-from icepyx.core.urls import CAPABILITIES_BASE_URL
+import harmony
 
 
 def get_capabilities(concept_id: str) -> dict[str, Any]:
-    response = requests.get(
-        CAPABILITIES_BASE_URL,
-        params={"collectionId": concept_id},
-    )
-    return response.json()
+    capabilities_request = harmony.CapabilitiesRequest(concept_id=concept_id)
+    # TODO: This will work if the user has a .netrc file available but the other
+    # auth options might fail. We might need to add harmony client auth to the
+    # icepyx auth package.
```

Comment:
icepyx.auth is a mixin for earthaccess, so it is easily made available wherever credentials are needed. I poked around harmony-py a bit and it looks like they don't allow you to just pass in a valid session, but I think earthaccess lets you just grab the username and password, so if we have the existing auth object we should be able to authenticate with harmony.Client that way.